cli/src/alluxio.org/cli/cmd/exec/class.go

```diff
@@ -0,0 +1,81 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package exec
+
+import (
+	"alluxio.org/cli/env"
+	"alluxio.org/log"
+	"bytes"
+	"github.com/palantir/stacktrace"
+	"github.com/spf13/cobra"
+)
+
+var Class = &ClassCommand{
+	BaseJavaCommand: &env.BaseJavaCommand{
+		CommandName:   "class",
+		JavaClassName: "",
+	},
+}
+
+type ClassCommand struct {
+	*env.BaseJavaCommand
+	mainClass string
+	jarFile   string
+	module    string
+}
+
+func (c *ClassCommand) Base() *env.BaseJavaCommand {
+	return c.BaseJavaCommand
+}
+
+func (c *ClassCommand) ToCommand() *cobra.Command {
+	cmd := c.Base().InitRunJavaClassCmd(&cobra.Command{
+		Use:   "class mainClass | --jar jarFile | --m module",
```

Comment:
omit the flags here since the flags will be already listed in the help text. maybe you can refer to the usage text of other commands